state_machine/applications/system-check/tests/burnwire_test.py

```diff
@@ -0,0 +1,86 @@
+"""
+Python system check script for PyCubed satellite board
+PyCubed Mini mainboard-v02 for Pocketqube Mission
+Burnwire Deployment Test
+* Author(s): Yashika Batra
+"""
+
+import time
+from lib import pycubed as cubesat
+
+
+def user_test():
+    """
+    All user interaction happens in this function
+    Prompt users, get user inputs for voltage level and duration and return
+    """
+
+    wait_time = 10
+
+    # conduct the test
+    print("Initializing burnwire test...")
+    print("You can test using a multimeter or fishing line.")
+    print("If using a multimeter, please read the voltage between the" +
+          "ground pin (GND) and burnwire pin on the -Z solar board.")
+
+    # get user input for voltage level and duration
+    voltage = float(input(
+        "At what voltage do you want to run the burnwire IC? (up to 3.3V): "))
+    burn_time = int(input(
+        "For how long do you want to run the burnwire test? "))
+    print("Test starting in {wait_time} seconds.")
```

Comment:
could we do a type (y) to run the burn wire test, (n) to cancel. Rather than a arbitrary 10 second wait time. I'm afraid the wait time could either be annoyingly long, or not enough time if things aren't fully set up yet.